testdata/e2e/installSealedSecretsWebChart.sh

```diff
@@ -7,6 +7,8 @@ helm upgrade --install sealed-secrets-web chart \
   -f testdata/e2e/e2e-values.yaml \
   --set format=${1} \
   --set sealedSecrets.certURL=${2} \
+  --set sealedSecrets.namespace=${3} \
+  --set sealedSecrets.serviceName=${4} \
```

Comment:
This makes no difference when the certURL

Reply:
https://github.com/bakito/sealed-secrets-web/pull/145/#discussion_r1192752611

> hence the ss.Namespace and ss.Service are empty and we get the error

where does setting the certUrl remove those values?